### ISSUE TITLE
New installation doesn't copy subfolders and files

### ISSUE THREAD

**Issue by dfranciscus** [chocolatey/simple-server#18]
Seems that the app_data and contents folder do not copy to c:\tools\chocolatey.server from the package. Anyone else seeing this?

**Comment by ferventcoder** [comment]
Can you submit the install.log? It has some detail about what it copies and what it skips.

**Comment by ferventcoder** [comment]
Submit - provide the detailed part of the log starting at the point where it hands over to the PowerShell script, the one from the log file will have debug data.

**Comment by ferventcoder** [comment]
If App_Data exists, it will not copy it again - this prevents overwriting of the logs and existing packages.

**Comment by ferventcoder** [comment]
Not sure on contents, but a guess is that we did not set recurse properly when there are no contents in the tools folder and it creates it all.

**Comment by ferventcoder** [comment]
Specifically this line: https://github.com/chocolatey/simple-server/blob/910b10669a12a14d1b308cdd9291188a5b731830/nuget/tools/chocolateyInstall.ps1#L50

**Comment by dfranciscus** [comment]
@ferventcoder where is install.log located?

**Comment by ferventcoder** [comment]
My bad 😆, I totally meant the chocolatey.log, did not mean to put that dot in between install and log.

**Comment by FLeven** [comment]
Same problem here, installed 2.0 using the same dockerfile I used for 0.1.3 and the Folders: App_Data, Bin, Content were missing and accessing the page ended in a runtime error. I have fixed it by manualy moving the missing folders to C:\tools\chocolatey.server inside the container and it works now.

**Comment by dfranciscus** [comment]
[chocolatey.log](https://github.com/chocolatey/simple-server/files/1611804/chocolatey.log)

@ferventcoder  here you go

**Comment by dfranciscus** [comment]
@FLeven same worked for me. For the time being I just added code into the Puppet module to copy those as a work around.

**Comment by ferventcoder** [comment]
~~~
Container cannot be copied onto existing leaf item.
Container cannot be copied onto existing leaf item.
~~~

Found the issue. Copy-Item won't create the top level folder if it doesn't exist (even if you use `-Force`).

**Comment by ferventcoder** [comment]
Fixed in 0.2.1
